public/app/plugins/panel/canvas/editor/LayerElementListEditor.tsx

```diff
@@ -198,6 +198,7 @@ export class LayerElementListEditor extends PureComponent<Props> {
 
     const onNameChange = (element: ElementState, name: string) => {
       element.onChange({ ...element.options, name });
+      this.onSelect(element);
```

Comment:
This works well, nice! In [Adela's PR](https://github.com/grafana/grafana/pull/52701) we are actually replacing this file with the `TreeViewEditor` so we may want to hold off until that gets merged in and then apply this fix to that file for this PR 👍

Reply:
Adela's PR has been merged 🎆, so we should be good to update this to point towards the `TreeNavigationEditor` file instead 👍

Reply:
After rebasing, it appears this fix is no longer needed, but please confirm:
![Jul-28-2022 00-04-59](https://user-images.githubusercontent.com/60050885/181442520-8ee6c918-9446-4607-a98d-f4765c7ad7c3.gif)

Reply:
It seems to work as it's supposed to. I haven't noticed any issues with it while working on my PR. 🎉